src/ElementProxy.js

```diff
@@ -12,26 +12,31 @@ import {NodeProxy} from './types'
 export class ElementProxy {
   _node: HTMLElement;
 
+  // Expose these so they can be stubbed for unit test
+  _emitMount: Function;
+  _emitUnmount: Function;
+
   constructor (node: HTMLElement) {
     this._node = node
   }
 
   emitMount (fn: Function): void {
-    emitMount(this._node, fn)
+    this._emitMount(this._node, fn)
   }
 
   emitUnmount (fn: Function): void {
-    emitUnmount(this._node, fn)
+    // Rely on prototype method so it can be stubbed for unit test
+    this._emitUnmount(this._node, fn)
```

Comment:
It was an attempt at letting the unit tests verify that `ElementProxy` is interacting with an underlying API so that we don't write the same tests for `ElementProxy` as we do for mountable. I think it is weak, but it is the best I came up with. What would you do?